payment_paysera/models/payment_transaction.py

```diff
@@ -79,11 +79,14 @@ def check_keys_not_empty(dict_, keys, example='SOME_VALUE'):
         params = data.get('params', {})
         check_keys_not_empty(params, ('projectid', 'requestid'))
 
-        the_data = data.get('data')
+        the_data = data.get('data', '')
 
-        ss1_received = data.get('ss1')
+        ss1_received = bytes(data.get('ss1'), 'ascii')
         ss1_computed = paysera.md5_sign(
-            the_data, bytes(self.acquirer_id.paysera_sign_password, 'ascii'))
+            bytes(the_data, 'ascii'),
+            bytes(self.acquirer_id.paysera_sign_password, 'ascii')
+        )
```

Comment:
it seems to me `md5_sign` function signature should just be `(string, string) -> string`, so that conversion operators wouldn't be required on all of the arguments / compared strings?